### ISSUE TITLE
Performance issue when lottie has png

### ISSUE THREAD

**Issue by sonphan12** [Samsung/rlottie#452]
`blend_image_xform` executing time is too long. 

Lottie resource: [performance.zip](https://github.com/Samsung/rlottie/files/5534554/performance.zip)

Test device: Huawei GR5 mini

Benchmarked on this lottie resource, it took about 40-70ms each time for `blend_image_xform` to be done. Airbnb's [lottie-android](https://github.com/airbnb/lottie-android) is a lot faster.

**Comment by smohantty** [comment]
@sonphan12 ,
yes the image implementation is not optimal. it was using Qt version before but was removed because of license issue. So i just added a simple version for feature completeness. I need to find out if there is some optimal implementation which is MIT license then we can make use of that. If you have some idea you can contribute as well :)

**Comment by sonphan12** [comment]
@smohantty 
Is it slow because of the matrix processing?

**Comment by sonphan12** [comment]
@smohantty 
Thanks for your response. As I library user with very little experiences in C++, I have no idea how to improve this :( But thank to your response, guess I'm gonna revert to the old pieces of code that use Qt to improve my app's performance of png lottie.

I'm gonna leave this issue open for now since it's still a known-but-not-solved-yet problem.

**Comment by sonphan12** [comment]
@smohantty 
Oh, and can you please tell me which piece of code belongs to Qt that you removed. I've checked in commit `6547245b`, seems like this commit is the one that removed Qt's part. But I can't figure out where are the Qt's codes.

**Comment by sonphan12** [comment]
@smohantty 
Sorry to bother you again but I have good news. Qt is now open for commercial usage

![image](https://user-images.githubusercontent.com/18615066/99244385-95b4da00-2834-11eb-8cae-8afac8f2fc18.png)

It would be awesome if you could update it back to rlottie, png performance should be improved for good :D

**Comment by sonphan12** [comment]
@smohantty 
I tried to revert to before commit removing Qt's code. It is still too slow.
